.github/pull_request_template.md

```diff
@@ -8,6 +8,6 @@ Fixes # (issue)
 
 ### Quick checks:
 
-- [ ] There is no other [pull request](https://github.com/conduitio/conduit-connector-connectorname/pulls) for the same update/change.
+- [ ] There is no other [pull request](https://github.com/github.com/maha-hajja/conduit-connector-weather/pulls) for the same update/change.
```

Comment:
This didn't work well... You were using the script right?

Reply:
Good catch, seems like we need to fix the script.

Reply:
yes

Reply:
I think we should also update the whole template, some configuration details are redundant after the `sdk.Util.ParseConfig` addition, and paramgen, but this could be done later ofc